data/validation.yml

```diff
@@ -19,3 +19,15 @@ questions:
         answers:
             - {value: "True",    correct: true}
             - {value: "False",   correct: false}
+    -
+        question: 'True or False ? We can also apply constraints on class getters with ``addPropertyConstrait()``'
+        answers:
+            - {value: "True",    correct: false}
+            - {value: "False",   correct: true}
+    -
+        question: 'True or False ? In Symfony:'
```

Comment:
why not ? I will update my pull request. Thank you for your interest, don't be shy, propose some new questions :)